### PR TITLE
rearrange dep work loop to try to regain some speed

### PR DESCRIPTION
**Pull request by dobbymoodge**

### CODE REVIEW

**Comment by dobbymoodge** [comment]
@danmcp This seems like it might be a bit faster. I dumped all the URIs hit during a run and no API calls are getting hit twice, so it looks like caching is working as intended. Spending a lot of time in `Trello::List#cards` and `Trello::Card#checklists`, so maybe it's just going to be slow now? :(

**Comment by danmcp** [comment]
@dobbymoodge That doesn't really make sense.  It was twice as slow and the job does a lot of other things.  How long does it take if you just have it process the docs board work still?

**Comment by dobbymoodge** [comment]
@danmcp I'll test that out in just a minute.

**Comment by dobbymoodge** [comment]
@danmcp Processing just the docs labels brings it down to 12 minutes, which is still 2x as long as before. Going to have to dive a bit deeper...

File: trello [hunk 1]
```diff
@@ -865,32 +865,32 @@ command :update do |c|
...
-                  if options.update_bug_tasks
-                    bugzilla = load_conf(BugzillaHelper, CONFIG.bugzilla, true)
-                    trello.update_bug_tasks(card, bugzilla)
-                  end
-                  if options.add_dependent_tasks && dep_work_board_params.include?(:task_reminder_text)
-                    trello.add_dependent_tasks_reminder(card, dep_work_board_params[:task_reminder_text], dep_work_board_params[:label])
-                  end
+                end
+                if options.update_bug_tasks
+                  bugzilla = load_conf(BugzillaHelper, CONFIG.bugzilla, true)
```
Comment (dobbymoodge) [inline trello]: oops, I'm instantiating the bugzilla object every iteration. I'll fix that server-side and see if that speeds things up.

**Comment by dobbymoodge** [comment]
@danmcp yeah, this little change fixes everything. :)

**Pull request merged**